src/controllers/birthday_message.py

```diff
@@ -44,15 +44,16 @@ def is_birthday(birthday: str):
         return [user.get(BambooIntegration.employee_identifier_field) for user in users if is_birthday(user.get(BambooIntegration.employee_birthday_field))]
```

Comment:
maybe we can check if birthday is not None, just to make sure

Reply:
Great, suggestion applied!